lacommunaute/forum_conversation/models.py

```diff
@@ -23,6 +25,18 @@ def get_absolute_url(self):
             },
         )
 
+    @property
+    def poster_email(self):
+        if self.first_post.username:
+            return self.first_post.username
+        return self.first_post.poster.email
```

Comment:
```suggestion
        return self.first_post.username or self.first_post.poster.email
```
Applicable normalement à `poster_display_name()` si fusion des fonctions _forum_member_display_name_.

Reply:
Great! 
Je n'ai pas fusionné `get_forum_member_display_name`  qui traite une instance `User` avec `get_anonymous_forum_member_display_name` qui traite un email dans une `string`, mais la propriété `poster_display_name` est plus légère avec ta suggestion.